### PR TITLE
Add target FLYWOOF405S_AIO AND FLYWOOF411_5IN1_AIO

### PR DESCRIPTION
**Pull request by flywoo**
Add target FLYWOOF405S_AIO AND FLYWOOF411_5IN1_AIO

https://flywoo.net/collections/aio-evo-fc/products/goku-gn405s-20a-aio-mpu6000-25-5-x-25-5

https://flywoo.net/collections/aio-evo-fc/products/goku-versatile-f4-5-in-1-aio

### CODE REVIEW

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Betaflight / STM32F411 (S411) 4.2.8 Feb 15 2021 / 12:09:04 (101738d8e) MSP API: 1.43
+
+board_name FLYWOOF4115IN1AIO
```
Comment (haslinghuis) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:3]:
```suggestion
board_name FLYWOOF411_5IN1_AIO
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
board_name does not match

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config (hunk 1 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF411_5IN1_AIO.config]: Thanks for the reminder, it has been edited

File: configs/default/FLWO-FLYWOOF405S_AIO.config [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 170
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405S_AIO.config:108]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+# feature
+feature -RX_PARALLEL_PWM
+feature LED_STRIP
+feature OSD
+feature RX_SPI
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:92]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set baro_hardware = NONE
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:95]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+set serialrx_provider = SBUS
+set rx_spi_protocol = FRSKY_D
+set rx_spi_bus = 3
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set dshot_bidir = ON
+set dshot_bitbang = AUTO
+set motor_pwm_protocol = DSHOT300
+set align_board_roll = 180
+set align_board_yaw = 180
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:105]: gyro alignment should not be correction by these

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 200
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:111]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+set max7456_spi_bus = 2
+set frsky_spi_tx_id = 64,120
+set frsky_spi_offset = -103
+set frsky_spi_bind_hop_data = 6,59,114,169,224,44,99,154,209,29,84,139,194,14,69,124,179,234,54,109,164,219,39,94,149,204,24,79,136,189,9,64,119,174,229,49,104,159,214,34,89,144,199,19,74,129,184,0,0,0
+set dashboard_i2c_bus = 1
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:116]: we don't support this

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 200
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set frsky_spi_tx_id = 64,120
+set frsky_spi_offset = -103
+set frsky_spi_bind_hop_data = 6,59,114,169,224,44,99,154,209,29,84,139,194,14,69,124,179,234,54,109,164,219,39,94,149,204,24,79,136,189,9,64,119,174,229,49,104,159,214,34,89,144,199,19,74,129,184,0,0,0
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:115]: Isn't these the default?

**Review by DusKing1 — CHANGES_REQUESTED** [review]

**Comment by flywoo** [comment]
@DusKing1 Thanks for the help, it's all fixed

**Comment by haslinghuis** [comment]
Please squash your commits.

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set rx_spi_protocol = FRSKY_D
+set rx_spi_bus = 3
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set dshot_bidir = ON
+set dshot_bitbang = AUTO
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:100]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+set battery_meter = ADC
+set ibata_scale = 200
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 2
+set frsky_spi_tx_id = 64,120
+set frsky_spi_offset = -103
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = DEFAULT
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:112]: default setting

File: configs/default/FLWO-FLYWOOF411_5IN1_AIO.config [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+feature LED_STRIP
+feature OSD
+feature RX_SPI
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF411_5IN1_AIO.config:94]: Isn't' this an AIO with SPI RX build-in? Why you need SBUS again?

**Review by DusKing1 — CHANGES_REQUESTED** [review]

**Comment by flywoo** [comment]
@DusKing1 Thanks for the help, it's all fixed

**Comment by flywoo** [comment]
@haslinghuis already squash commits thank

File: configs/default/FLWO-FLYWOOF405S_AIO.config (hunk 2 above)
Comment (haslinghuis) [inline configs/default/FLWO-FLYWOOF405S_AIO.config:108]: Please remove `system_hse_mhz` as it's default

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/FLWO-FLYWOOF405S_AIO.config (hunk 2 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @haslinghuis Edited, sorry for omission. thank you

Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @haslinghuis already squash commits thank

Comment (haslinghuis) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @DusKing1 all good here? Or do we need to to change SBUS to SPI?

Comment (DusKing1) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: IDK, I don't have any hardware sample

Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @DusKing1 Our default is the SPI receiver.

Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: Tested the code, no problem

Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @haslinghuis @DusKing1 The code has been tested and there is no problem. The current default is the SPI FRSKY_D receiver protocol.

Comment (DusKing1) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @flywoo I have no idea what you are talking about

Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @DusKing1 Is there any problem with the target ? We have passed the test. thank you for your help.

Comment (DusKing1) [reply configs/default/FLWO-FLYWOOF405S_AIO.config]: @flywoo I have no idea what you are talking about

**Comment by flywoo** [comment]
@haslinghuis The target can merged commit ? Please let me know if any modification is required, thanks

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**